templates/withdraw/withdraw-dashboard.php

```diff
@@ -0,0 +1,108 @@
+            <div class="dokan-w8">
+                <p>
+                    <strong><?php esc_html_e( 'Last Payment', 'dokan-lite' ); ?></strong><br>
+                    <?php echo wp_kses_post( $payment_details ); ?>
+                </p>
+
+                <?php do_action( 'dokan_withdraw_content_after_last_payment' ); ?>
+
+            </div>
+            <div class="dokan-w5">
+                <a href="<?php echo esc_url( dokan_get_navigation_url( 'withdraw-requests' ) ); ?>" class="dokan-btn" id="dokan-withdraw-display-requests-button"><?php esc_html_e( 'View Payments', 'dokan-lite' ); ?></a>
+                <?php do_action( 'dokan_withdraw_content_after_last_payment_button' ); ?>
+            </div>
+        </div>
+
+        <?php do_action( 'dokan_withdraw_content_after_last_payment_section' ); ?>
+
+    </div>
+</div>
+
+<?php do_action( 'dokan_withdraw_content_after_payment_details_section' ); ?>
+
+
+<div class="dokan-panel dokan-panel-default" id="dokan-withdraw-payment-method-list" data-security="<?php echo esc_attr( wp_create_nonce( 'dokan_withdraw_make_default' ) ); ?>" >
+    <div class="dokan-panel-heading"><strong><?php esc_html_e( 'Payment Methods', 'dokan-lite' ); ?></strong></div>
+    <div class="dokan-panel-body general-details">
+        <?php
+        foreach ( $active_methods as $method ) :
+            $method_icon     = dokan_withdraw_get_method_icon( $method );
+            $method_info     = dokan_withdraw_get_method_additional_info( $method );
+            $method_title    = dokan_withdraw_get_method_title( $method );
+            $has_information = in_array( $method, dokan_get_seller_active_withdraw_methods() );
+            ?>
+            <div class="dokan-clearfix dokan-panel-inner-container">
+                <div class="dokan-w8">
+                    <img src="<?php echo esc_url( $method_icon ); ?>" width="40" height="40" alt="<?php echo esc_attr( $method_title ); ?>" class="dokan-withdraw-method-logo" />
+                    <strong><?php echo esc_html( $method_title ); ?></strong>
+                    <small><?php echo esc_html( $method_info ) ?></small>
+                </div>
+                <div class="dokan-w5">
+                    <?php if ( $default_method === $method ) : ?>
+                    <button class="dokan-btn dokan-btn-default" disabled data-method="<?php echo esc_attr( $method ); ?>"><?php esc_html_e( 'Default', 'dokan-lite' ); ?></button>
+                    <?php elseif ( ! $has_information ) : ?>
+                    <a href="<?php echo esc_url( dokan_get_navigation_url( 'settings/payment' ) ); ?>" class="dokan-btn" data-method="<?php echo esc_attr( $method ); ?>"><?php esc_html_e( 'Setup', 'dokan-lite' ); ?></a>
+                    <?php else: ?>
+                   <button class="dokan-btn dokan-withdraw-make-default-button" data-method="<?php echo esc_attr( $method ); ?>"><?php esc_html_e( 'Make Default', 'dokan-lite' ); ?></button>
```

Comment:
Kindly fix the alignment here:

<img width="401" alt="Screenshot 2021-11-05 at 11 15 12 AM" src="https://user-images.githubusercontent.com/59737258/140461887-8978c268-a6ef-4459-a856-ee44c4b02373.png">